.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
       "language": "svelte",
       "autoFix": true
     }
-  ]
+  ],
+  "editor.codeActionsOnSave": {
```

Comment:
I'm pretty sure we should move .vscode to gitignore. It's a user-specific setting. What do you think?

Reply:
I tend to commit these, personally, but it might be a little controversial.  It's helpful for me when I switch between machines and for my colleagues where we all use the same IDE.